.github/workflows/run-tests.yml

```diff
@@ -42,13 +47,22 @@ jobs:
 
     steps:
     - uses: actions/checkout@v2
+    - name: Cache chianti database
+      uses: actions/cache@v2
+      if: matrix.toxenv != 'build_docs' && matrix.toxenv != 'codestyle'
+      env:
+        # Bump the version number when changing the chianti version to test with
+        cache-name: cache-chianti-dbase-807
+      with:
+        path: ~/.chianti
+        key: ${{ env.cache-name }}-${{ matrix.toxenv }}
```

Comment:
And this is the key that it's cached based on, ie. if this key changes then the old cache is deleted and replaced by a new version. I've just set it to a custom name, and the name of the tox environment, as the chianti database only changes when the chianti version changes.